flight4handler.go

```diff
@@ -128,6 +129,17 @@ func flight4Parse(ctx context.Context, c flightConn, state *State, cache *handsh
 		cfg.writeKeyLog(keyLogLabelTLS12, clientRandom[:], state.masterSecret)
 	}
 
+	if len(state.SessionID) > 0 {
+		s := Session{
+			ID:     state.SessionID,
+			Secret: state.masterSecret,
+			Addr:   c.RemoteAddr().String(),
+		}
+		cfg.sessionStore.Set(&s, false)
+
+		cfg.log.Tracef("[handshake] save session: %+v", s)
```

Comment:
Secrets must not be exposed to the standard log.

Reply:
I will drop this trace log.